lang/java/reef-applications/reef-vortex/src/main/java/org/apache/reef/vortex/examples/hello/HelloVortexFunction.java

```diff
@@ -30,7 +28,7 @@
    * Prints to stdout.
    */
   @Override
-  public Serializable call(final Serializable serializable) throws Exception {
+  public Object call(final Object object) throws Exception {
```

Comment:
There is an `AddOne` and `MatMul` which use the typed input and output. Or as the input/output don't have to be `Serializable`, they could be `Void` same as they used to be originally. How does it sound?